### PR TITLE
ANDROAPP-5617-mobile-ui-Horizontal-RadioButoons-and-checkboxes-resize-when-clear-button-appears

### PR DESCRIPTION
**Pull request by DavidAparicioAlbaAsenjo**

### CODE REVIEW

File: designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/InputRadioButton.kt [hunk 1]
```diff
@@ -90,6 +92,8 @@ fun InputRadioButton(
                         onItemChange.invoke(null)
                     },
                 )
+            } else {
+                Spacer(modifier = Modifier.width(Spacing.Spacing24))
             }
```
Comment (siddh1004) [inline designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/InputRadioButton.kt:97]: @DavidAparicioAlbaAsenjo This component still change it's space. I think we need to update the Size to 48 as the size of icon is 48 too. 
image
image

File: designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/InputRadioButton.kt [hunk 2]
```diff
@@ -90,6 +92,8 @@ fun InputRadioButton(
                         onItemChange.invoke(null)
                     },
                 )
+            } else {
+                Spacer(modifier = Modifier.width(Spacing.Spacing24))
```
Comment (siddh1004) [inline designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/InputRadioButton.kt:96]: `Spacer(modifier = Modifier.width(Spacing.Spacing48))`

File: designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/InputCheckBox.kt [hunk 3]
```diff
@@ -85,6 +87,8 @@ fun InputCheckBox(
                         onClearSelection.invoke()
                     },
                 )
+            } else {
+                Spacer(modifier = Modifier.width(Spacing.Spacing24))
```
Comment (siddh1004) [inline designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/InputCheckBox.kt:91]: `Spacer(modifier = Modifier.width(Spacing.Spacing48))`

File: designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/InputCheckBox.kt [hunk 4]
```diff
@@ -85,6 +87,8 @@ fun InputCheckBox(
...
             }
```
Comment (siddh1004) [inline designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/InputCheckBox.kt:92]: @DavidAparicioAlbaAsenjo This component still change it's space. I think we need to update the Size to 48 as the size of icon is 48 too. 
image
image

File: designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/InputRadioButton.kt (hunk 2 above)
Comment (siddh1004) [inline designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/InputRadioButton.kt:96]: @DavidAparicioAlbaAsenjo This component still change it's space. I think we need to update the Size to 48 as the size of icon is 48 too. 
image
image

**Review by siddh1004 — CHANGES_REQUESTED** [review]

**Review by andresmr — APPROVED** [review]

**Review by siddh1004 — APPROVED** [review]

**Pull request merged**